internal/cli/get.go

```diff
@@ -318,7 +318,7 @@ func printDevicesTable(w *tabwriter.Writer, response *apiclient.ListDevicesRespo
 }
 
 func printEnrollmentRequestsTable(w *tabwriter.Writer, response *apiclient.ListEnrollmentRequestsResponse) {
-	fmt.Fprintln(w, "NAME\tAPPROVAL\tAPPROVER\tAPPROVED LABELS")
+	fmt.Fprintln(w, "FINGERPRINT\tAPPROVAL\tAPPROVER\tAPPROVED LABELS")
```

Comment:
we store this value in the db as `name`  should we align on a naming standard for the product.

Reply:
That's a good point. We should be consistent also at the API / server level.

Reply:
We should align everything.  Do we currently use the "fingerprint" as a name, and later might use something else as a name (e.g., hostname)?  Then maybe we have name and fingerprint, both with the same value currently?  Or do we just have name?  Or just fingerprint?
@fzdarsky

Reply:
FYI, in the UI we have:
- "Fingerprint --> Backend's "name"
- "Name" --> based on the "displayName" label (optional)

Reply:
Note that in general, `$resource.metadata.name` is an identifier and immutable. The agent currently sets this to its fingerprint, but that may change (e.g. allowing users to configure the agent to generate its name based on its MAC address or similar instead). So +1 to using "Name" everywhere.

Reply:
I would probably rename "name" to "ID", and then use a label for displayname.

Reply:
`$resource.metadata.name` is what we use for K8s-compatibility and tools like kustomize to work on our manifests, so that shouldn't change. Do you mean just in the column header of the CLI/UI?

Reply:
Right, resource.metadata.name is the resource identifier that doesn't change. We can call this ID in the UI, though that might be confusing. Maybe support a displayName label, and we'll have name vs displayName?

Reply:
After further discussion, we agreed that:
- The Backend and CLI will stay as is: the main identifier is called "name", and it currently is populated with the device fingerprint
- The UI columns will change: "Name" will be renamed to "Alias", and  "Fingerprint" will be renamed to "Name".

This allows that everywhere in Flight Control we use the same terminology and are aligned in this matter.

I'll close this MR and create a new one in the UI for the aforementioned changes.